CREDITS

```diff
@@ -1,3 +1,3 @@
 geos
-Sandro Santilli <strk@kbt.io> - GEOS
-J Smith <dark.panda@gmail.com> <jay@php.net> - PHP packaging
+Sandro Santilli strk@kbt.io - GEOS
+J Smith dark.panda@gmail.com jay@php.net - PHP packaging
```

Comment:
why this format change ?

Reply:
Because of < > The RC resource compiler crashes

Reply:
sounds like an upstream bug. The current one does not crash, right ?

Reply:
yes the current version compiles without errors